vis.mac

```diff
@@ -81,8 +81,8 @@
 
 # Vis Settings for SK
 /vis/viewer/zoom 1.2
-#/vis/viewer/set/viewpointThetaPhi 45 45 deg
-/vis/viewer/set/viewpointThetaPhi 110 0 deg
+/vis/viewer/set/viewpointThetaPhi 45 45 deg
+#/vis/viewer/set/viewpointThetaPhi 110 0 deg
```

Comment:
vis.mac is still showing up in the diff, I think because you have the wrong /vis/viewer/set/viewpointThetaPhi uncommented. Make sure the 45 45 one is commented and the 110 0 is uncommented (opposite to what you currently have here).